server/feed.js

```diff
@@ -0,0 +1,49 @@
+var soundcloud = require('../soundcloud/api-client');
+var knexfile = require('../knexfile');
+var knex = require('knex')(knexfile);
+var _ = require('lodash');
+var Promise = require('bluebird');
+
+module.exports = function fetchFeedApi(soundcloudUserId, token, nextLink) {
+    return Promise.join(
+        fetchSoundcloudFeed(token, nextLink),
+        fetchBlacklist(soundcloudUserId),
+        function (feed, blacklist) {
```

Comment:
Hu ... Promise.join doesn't work like that afaik. You are using it like you're in nuclear-js ...